Files/UserControls/NavigationToolbar.xaml

```diff
@@ -858,40 +858,35 @@
                     <DataTemplate x:DataType="filesystem:ListedItem">
                         <Grid Tag="{x:Bind ItemPath}">
                             <StackPanel Orientation="Horizontal" Spacing="8">
-                                <Grid
+								<Grid
```

Comment:
Can you fix the formatting here?

Reply:
I tried multiple times, but can't get this one to look right on GitHub without the VS extension.

Reply:
Can you try with the extension? I think you can configure it to require a manual action to run it, this should resolve the issue with it conflicting with your other work.